main.bib

```diff
@@ -1148,4 +1148,10 @@ @article{bieman1995cohesion
   pages={259--262},
   year={1995},
   publisher={ACM New York, NY, USA}
-}
\ No newline at end of file
+}
+
+@article{curtis1979,
+  author = {Curtis, Bill},
+  title = {Program Complexity and Software Errors: A Front End for Reliability},
+  year = {1979}
```

Comment:
@awggeez if it's an `@article`, let's add its `journal` and also `doi`

Reply:
the record type was changed and the corresponding fields were added